server/platform/services/sharedchannel/sync_send_remote.go

```diff
@@ -566,7 +573,7 @@ func (scs *Service) sendSyncData(sd *syncData) error {
 }
 
 // sendUserSyncData sends the collected user updates to the remote cluster.
-func (scs *Service) sendUserSyncData(sd *syncData) error {
+func (scs *Service) sendUserSyncData(sd *syncData, latestTimestamp int64) error {
```

Comment:
WDYT about making `latestTimestamp` part of the `syncData` struct and making it more explicit that it belongs to the global user sync?

I'm thinking about keeping the signature as is, and having a `syncData.GlobalUserSyncLastTimestamp` for that same purpose